packages/config/package.json

```diff
@@ -2,10 +2,12 @@
   "name": "@truffle/config",
   "version": "1.1.21",
   "description": "Utility for interacting with truffle-config.js files",
-  "main": "index.js",
-  "typings": "./typings/index.d.ts",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
```

Comment:
Oooh, this looks like our error. This property should be `"types"` and not `"typings"`.

Reply:
For reference https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html

Reply:
These are synonyms, as vscode says, but I’d better leave types to not cause questions.

![types](https://user-images.githubusercontent.com/10329528/65815343-0ba44f00-e1f7-11e9-930e-16133d16aca8.png)